data/db_demo.yaml

```diff
@@ -195,6 +195,7 @@ instrument:
     filters: ["ztfg", "ztfr", "ztfi"]
     =id: ZTF
     api_classname: ZTFAPI
+    api_observationplan_classname: ZTFMMAAPI
     # field_data:
     #   file: ../data/ZTF_Fields.csv
     # field_region:
```

Comment:
I think its nice to have because they at least show people how to use the field data when they want it, but it was commented out because it makes the demo data take too long, so I think this goes both ways.

Reply:
I think this is fine. I wasn't sure if this interface was still relevant.